stdlib/public/Concurrency/Executor.swift

```diff
@@ -461,6 +461,14 @@ public protocol TaskExecutor: Executor {
   // work-scheduling operation.
   @_nonoverride
   func enqueue(_ job: consuming ExecutorJob)
+
+  #if !$Embedded
+  // Must implement this requirement in order
+  @_nonoverride
+  @available(StdlibDeploymentTarget 6.2, *)
+  func enqueue(_ job: consuming ExecutorJob, isolatedTo serialExecutor: UnownedSerialExecutor/*some SerialExecutor*/)
```

Comment:
Should there be an implementation of this in `DispatchExecutor.swift`? It's not obvious to me what the `isolatedTo:` argument actually does here, or how an executor implementation would use it. Maybe some doc comments might help?